log4j-api/src/main/java/org/apache/logging/log4j/status/StatusLogger.java

```diff
@@ -158,11 +162,26 @@ public class StatusLogger extends AbstractLogger {
 
     /**
      * The name of the system property that can be configured with a {@link java.time.format.DateTimeFormatter} pattern that will be used while formatting the created {@link StatusData}.
+     * <p>
+     * When not provided, {@link Instant#toString()} will be used.
+     * </p>
      *
+     * @see #STATUS_DATE_FORMAT_ZONE
      * @since 2.11.0
      */
     public static final String STATUS_DATE_FORMAT = "log4j2.StatusLogger.DateFormat";
 
```

Comment:
Piotr's comment below also applies to this property name.

Reply:
I've pushed a fix for this. Could you check it again, please?